k8s/helmfile/env/staging/wbaas-backup.values.yaml.gotmpl

```diff
@@ -1,6 +1,9 @@
 image:
   tag: v0.3.1
 
+job:
```

Comment:
it fine but this isn't necessary since it would inherit from prod

Reply:
ah true!